### PR TITLE
Feature/edit container

### PR DESCRIPTION
**Pull request by deonjo**
Screenshot 2019-03-20 11 02 46
Screenshot 2019-03-20 11 02 32

### CODE REVIEW

File: src/reducers/real.jsx [hunk 1]
```diff
@@ -143,18 +143,13 @@ const realReducer = (state = initialState, action) => {
             return deleteContainerState
         }
             
-        case CONTAINER_RENAME: {
-            let containers = state.containers
-            let container = containers.find(ele => ele._id === action.container._id)
+        case CONTAINER_EDIT: {
```
Comment (jeffchiucp) [inline src/reducers/real.jsx]: for CONTAINER_EDIT, is THIS a new reducer and we are replacing CONTAINER_RENAME?

File: src/reducers/real.jsx [hunk 2]
```diff
@@ -143,18 +143,13 @@ const realReducer = (state = initialState, action) => {
...
+            const container = state.containers.find(ele => ele && ele._id === action.container._id)
```
Comment (jeffchiucp) [inline src/reducers/real.jsx]: so we are not using .filter, but just keeping .find?

File: src/reducers/real.jsx (hunk 1 above)
Comment (deonjo) [reply src/reducers/real.jsx]: Yes i am renaming the reducer, and i simplified it.

Comment (jeffchiucp) [reply src/reducers/real.jsx]: this is new action

File: src/components/container/container.jsx [hunk 3]
```diff
@@ -29,6 +32,54 @@ const styles = theme => ({
 })
 
 export class Container extends Component {
+    constructor(props) {
```
Comment (jeffchiucp) [inline src/components/container/container.jsx]: i see. Thanks for adding the constructors for the props

File: src/reducers/real.jsx (hunk 1 above)
Comment (jeffchiucp) [reply src/reducers/real.jsx]: i see. You Changed name from Rename Item to Edit, to be consistent with container menu

**Pull request merged**